examples/hidden_code_lines.md

```diff
@@ -0,0 +1,111 @@
+---
```

Comment:
This presentation was primarily for demonstration purposes, let me know if you think I should remove/amend it @mfontanini.

Reply:
This is great but for now I'd like to remove it. I usually write docs about new features along when I'm about to release the next version, and I'll use some of these examples there. I'm planning on, besides having docs for this (and snippet code execution in general, touching the `examples/code.md` example presentation to include more cases like these ones.

Reply:
Removed.